pandas/io/spss.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+from typing import Optional, Sequence, Union
+
+from pandas.api.types import is_list_like
+from pandas.core.api import DataFrame
+
+
+def read_spss(path: Union[str, Path],
+              usecols: Optional[Sequence[str]] = None,
+              convert_categoricals: bool = True) -> DataFrame:
+    """
+    Load an SPSS file from the file path, returning a DataFrame.
+
+    .. versionadded 0.25.0
+
+    Parameters
+    ----------
+    path : string or Path
+        File path
+    usecols : list-like, optional
+        Return a subset of the columns. If None, return all columns.
+    convert_categoricals : bool, default is True
+        Convert categorical columns into pd.Categorical.
+
+    Returns
+    -------
+    DataFrame
+    """
+    try:
```

Comment:
Apologies for the merge conflict, I just refactored this. You should be able to replace this try / except with `import_optional_dependency` from `pandas.compat._optional`:

http://pandas-docs.github.io/pandas-docs-travis/development/contributing.html#optional-dependencies

Can you pick a minimum version for pyreadstat that we should require? Add that to the `VERSIONS` dict in `pandas.compat._optional`.

Reply:
Done, except for there is a problem with different version tags in pyreadstat: https://github.com/Roche/pyreadstat/issues/28

Do you want to wait if @ofajardo has time to cut a new release where this is fixed (will be 0.2.6 then), or should we remove the minimum requirement for now?